src/compas_ags/diagrams/formdiagram.py

```diff
@@ -22,6 +22,8 @@ def __init__(self):
         })
         self.update_default_vertex_attributes({
             'is_fixed': False,
+            'fix_x': False,
+            'fix_y': False,
```

Comment:
we tend to use the convention that attributes with boolean values have an `is_` prefix...

Reply:
changed

Reply:
now they are 'is_fixed_x' and 'is_fixed_y'